src/test/java/tests/URLTest.java

```diff
@@ -28,8 +28,9 @@ public URLTest(List<File> testFiles) {
                 org.checkerframework.checker.https.HTTPSChecker.class,
                 "url",
                 "-Anomsgtext",
-                "-Astubs=stubs/",
-                "-nowarn");
+                "-Astubs=Stubs/",
```

Comment:
Is there a reason you changed the directory to "Stubs" from "stubs"? On unix it's generally considered better style to use lowercase for directory and file names.